rbs/MethodTypeTranslator.h

```diff
@@ -6,23 +6,38 @@
 
 namespace sorbet::rbs {
 
+struct RBSArg {
+    core::LocOffsets loc;
+    rbs_ast_symbol_t *name;
+    rbs_node_t *type;
+};
```

Comment:
It doesn't look like this type leaks out through the public interface, would you mind nesting it under the `private` section of `MethodTypeTranslator`?